frontend/src/features/admin/access-request/AccessRequestForm.tsx

```diff
@@ -50,7 +50,7 @@ export const AccessRequestForm: React.FunctionComponent<IAccessRequestFormProps>
         <SectionField label="IDIR/BCeID" wideScreen>
           <Input
             field="businessIdentifierValue"
-            placeholder={initialValues?.user?.businessIdentifierValue}
+            placeholder={initialValues?.businessIdentifierValue}
```

Comment:
typing bug

Reply:
There are a couple of these. Just to be sure, did you check that the ts model is correct?

Reply:
yes, these match the FormModel. Before initialValues was typed as any so TS wouldn't complain...

Reply:
You can also see that the snapshots now properly populate the placeholder with values - before they were empty as those properties didn't exist